docs/guides/operations-guide/network.md

```diff
@@ -4,6 +4,44 @@ sidebar_label: Network
 
 # Network
 
+## Modifying the network configuration
+
+While the network layout of a production environment is mostly static,
+sometimes it is necessary to make changes. There is however no guarantee that
+the transition from one network configuration state to the next does not cause
+interruptions to existing network flows. E.g. even very short down/up cycles of
+interfaces may cause additional disturbances to SDN networks in upper layers,
+which are not reconciled immediately or never at all.
+Therefore it is required to move any existing payload when modifying the
+network configuration of compute nodes. You may use the
+[OpenStack Resource Manager](./tools/resource-manager#live-migration)
```

Comment:
Looks like this link is broken?

Reply:
I forgot to adapt the link to the new location after copy&paste. I have removed it now, as it does not add any value to the documentation anyway and also seems to be outdated.  The resource manager has been included in `osism`, right?